pkg/controller/grafana/kubeHelper.go

```diff
@@ -30,9 +30,16 @@ func newKubeHelper() *KubeHelperImpl {
 	return helper
 }
 
-func (h KubeHelperImpl) getMonitoringNamespaces() ([]v1.Namespace, error) {
+func (h KubeHelperImpl) getMonitoringNamespaces(ls *metav1.LabelSelector) ([]v1.Namespace, error) {
+	s, err := metav1.LabelSelectorAsSelector(ls)
+	if err != nil {
+		return nil, err
+	}
+
+	var ss string
+	metav1.Convert_labels_Selector_To_string(&s, &ss, nil)
 	selector := metav1.ListOptions{
-		LabelSelector: "monitoring=enabled",
+		LabelSelector: ss,
```

Comment:
@sedroche for readability, probably better to have something like `selector`  and `selectorString` instead of `s` and `ss`